test/mlj_interface.jl

```diff
@@ -0,0 +1,60 @@
+
+@testset "MLJ Interface" begin
+    using CatBoost.MLJCatBoostInterface: CatBoostRegressor, CatBoostClassifier
+
+    @testset "MLJ Examples" begin
+        for ex in readdir(MLJ_EXAMPLES_DIR)
+            @testset "$ex" begin
+                # Just check the examples run, for now.
+                include(joinpath(MLJ_EXAMPLES_DIR, ex))
+            end
+        end
+    end
+
+    @testset "CatBoostClassifier" begin
+        X = DataFrame(; a=[1, 4, 5, 6], b=[4, 5, 6, 7])
+        y = [0, 0, 1, 1]
```

Comment:
You are generating a warning here because `scitype(y) = AbstractVector{Count}` but your implementation declares `target_scitype(y)` = `AbstractVector{<:Finite}` (which makes sense because your model is a classifier). 

So you probably want to do `y = coerce([0, 0, 1, 1], OrderedFactor)` or `y = coerce([0, 0, 1, 1], Multiclass)` here.

Reply:
Thank you for explaining this.